doc/source/getting-started/administration.rst

```diff
@@ -9,7 +9,7 @@ First step
 ----------
 
 To trigger the public release process, project leads must first complete the
-`Release request workflow initiation form <https://ansys.typeform.com/ReleaseSupport?typeform-source=gz2idtcjsw2.typeform.com>`_.
+`Release request workflow initiation form <open-source request form>`_.
```

Comment:
And make sure that the links are included into this file

Reply:
@jorgepiloto based on the error you are getting, my feeling is that the link is not placed properly or substituted

Reply:
Fixed!